src/commands/synthetics/reporters/default.ts

```diff
@@ -360,7 +360,7 @@ export class DefaultReporter implements MainReporter {
     )
   }
 
-  public testTrigger(test: Test, testId: string, executionRule: ExecutionRule, config: ConfigOverride) {
+  public testTrigger(test: Pick<Test, 'name'>, testId: string, executionRule: ExecutionRule, config: ConfigOverride) {
```

Comment:
Why this change?

Reply:
So I could call `defaultReporter.testTrigger` from a Jest test without building a whole `Test` object. In this method, we only use the name of the test, so requiring a whole object seems unnecessary IMO 🙂 (I believe this is called the "principle of least power"?)

I didn't change the interface definition, so custom reporters can still use the whole object, if need be.